app/jobs/stockit_delete_job.rb

```diff
@@ -1,13 +1,17 @@
 class StockitDeleteJob < ActiveJob::Base
   queue_as :stockit_updates
 
-  def perform(inventory_number)
-    response = Stockit::Browse.new(inventory_number).remove_item
+  def perform(package_id)
+    package = Package.find_by(id: package_id)
```

Comment:
@steveyken Previously, we were using `inventory_number` because we are calling this job while deleting package.
Sidekiq is fast enough that it will perform this job before the deletion of package.
But in case, if this job queued and tries to execute after the package is deleted, then this might not delete `item` from stockit.

Please let me know your thoughts. Thanks :)